ReSharper.FSharp/src/FSharp.Psi/src/Impl/Tree/Attribute.cs

```diff
@@ -10,5 +13,26 @@ internal partial class Attribute
       new CtorReference(this);
 
     public override IFSharpIdentifier FSharpIdentifier => ReferenceName?.Identifier;
+
+    public IList<IArgument> Arguments
+    {
+      get
+      {
+        // todo: this is same as NewExpr.Arguments
```

Comment:
We can try adding a common interface for `Attribute` and `NewExpr` to the grammar, and then define a util method extracting this logic and taking either of the two types as an argument.

Reply:
I've abstracted this to a util module